resources/careconnect-to-ukcore/allergyintolerance/input/AllergyIntoleranceDescriptionId-Extension-3to4_000.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Filename should be AllergyIntoleranceExtension-coding-sctdescid-3to4_002.json (002 as there is a 000 and 001 file already.)

This needs to match the map name, i.e. AllergyIntoleranceExtension-coding-sctdescid-3to4.map